### DISCUSSION TITLE
v3.0.3

### DISCUSSION THREAD

**Discussion by assafelovic** [assafelovic/gpt-researcher#720] [Announcements]
Excited to release our latest innovation in multi agent long report collaboration! Thanks to @DandinPower we've made progress in building long reports based on parallel agent work on a given topic with minimal redundancy in report quality. This is another step in building the leading AI research agent. Thanks to the amazing community for further pushing boundaries!

## What's Changed
* Async fix by @ruzicka02 in https://github.com/assafelovic/gpt-researcher/pull/705
* Add PubMed Central retriever by @0x11c11e in https://github.com/assafelovic/gpt-researcher/pull/711
* Feature: Reducing redundancy problems in Detailed Reports by @DandinPower in https://github.com/assafelovic/gpt-researcher/pull/713

## New Contributors
* @DandinPower made their first contribution in https://github.com/assafelovic/gpt-researcher/pull/713

**Full Changelog**: https://github.com/assafelovic/gpt-researcher/compare/v3.0.2...v3.0.3

<hr /><em>This discussion was created from the release <a href='https://github.com/assafelovic/gpt-researcher/releases/tag/v3.0.3'>v3.0.3</a>.</em>

**Comment by Josephrp** [comment]
super cool
